src/time.js

```diff
@@ -9,6 +9,20 @@ function advanceBlock () {
   });
 }
 
+// Advance the block to the passed height
+async function advanceBlockTo (target) {
+  if (!BN.isBN(target)) {
+    target = new BN(target);
+  }
+
+  let now = (await latestBlock());
+  if (target.lt(now)) throw Error(`Cannot advance current block (${now}) to a moment in the past (${target})`);
+  for (;target.gte(now);) {
```

Comment:
I'd rather avid yoda conditions :sweat_smile: 

```suggestion
  while (now.lt(target)) {
```

Reply:
Note that this can also be simplified by removing the `now` variable entirely and using:

```js
  while ((await latestBlock()).lt(target)) {
```